Assignment 2 - Menu/src/assignment/pkg2/menu/Assignment2Menu.java

```diff
@@ -0,0 +1,90 @@
+/*
+Dylan Tkach
+Java Programming
+CPT-163-27
+ */
+package assignment.pkg2.menu;
+
+import java.util.Scanner;
+
+public class Assignment2Menu {
+
+    public static void main(String[] args) {
+    
+        Scanner keyboard = new Scanner(System.in);
+        
+        int MenuSelection;
+        
+        int charW = 'W';
+        int charA = 'A';
+        int charR = 'R';
+        int charH = 'H';
+        // A declared.
+        int charM = 'M';
+        // M declared.
+        int charE = 'E';
+        // R declared.
+        
+        int total = charW + charA + charR + charH + charA + charM + charM + charE + charR;
+        
+        
+        System.out.println("|=====================================|");
+        System.out.println("| Please choose one of the following: |");
+        System.out.println("|                                     |");
+        System.out.println("| 1. Word Counter                     |");
+        System.out.println("| 2. Is it heresy?                    |");
+        System.out.println("| 3. List of Primarchs                |");
+        System.out.println("| 4. Frame of Reference               |");
+        System.out.println("| 5. Contact Info                     |");
+        System.out.println("| Enter 0 to exit.                    |");
+        System.out.println("|_____________________________________|");
```

Comment:
:joy_cat: :boom: I really like this. You can also do this with a concatenation symbol `+` and reduce the runtime complexity of the code.

Each time we run System.out...("thing") We have to execute some other code. If this was one big string though the compiler can combine them at compile time and then we would only need to output that single string once.

The compiler can automatically combine strings as long as they are literals. _Between the " " chars_

Here is an example of a lighter version that should operate the same.

```Java
System.out.println("|=====================================|\n"
                  +"| Please choose one of the following: |\n"
                  +"|                                     |\n"
                  +"| 1. Word Counter                     |\n"
                  +"| 2. Is it heresy?                    |\n"
                  +"| 3. List of Primarchs                |\n"
                  +"| 4. Frame of Reference               |\n"
                  +"| 5. Contact Info                     |\n"
                  +"| Enter 0 to exit.                    |\n"
                  +"|_____________________________________|");
```